src/Configuration/Cache.php

```diff
@@ -30,15 +31,15 @@ class Cache extends ConfigurationAnnotation
      * The number of seconds that the response is considered fresh by a private
      * cache like a web browser.
      *
-     * @var int
+     * @var int|string
```

Comment:
Why do we suddenly allow a string here?

Reply:
Because it's possible for smaxage, maxage and maxstale attributes to set string with relative time format (1 day, 2 weeks, …)
https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/cache.html

Reply:
Oh, so the `@var` was lying to us the whole time. Got it. :-)